src/metabase/driver/sql_jdbc/execute.clj

```diff
@@ -714,6 +712,29 @@
               results-metadata {:cols (column-metadata driver rsmeta)}]
           (respond results-metadata (reducible-rows driver rs rsmeta qp.pipeline/*canceled-chan*))))))))
 
+(defn sql->reducible-rows
```

Comment:
how is this different from `jdbc/reducible-query`?

Reply:
I've added a docstring and renamed the function to make the purpose a little clearer, and also nod towards it being similar to `jdbc/reducible-query`
https://github.com/metabase/metabase/blob/2d5975e9ea8d3d1ee7af755be34d9340cc06fbc5/src/metabase/driver/sql_jdbc/execute.clj#L717-L720

Reply:
I'm not sure about the name `simple-reducible-query`, but I wanted to distinguish it as the simpler version of `execute-reducible-query` in the same namespace.

Reply:
why not call it `reducible-query`? 

and I would mention the difference with `jdbc/reducible-query` in the docstring, I'm sure people will have the same question as I did.

Reply:
> why not call it reducible-query?

I wanted to distinguish it as the simpler version of `execute-reducible-query` in the same namespace. That's not so obvious though. I will accept your suggestion :)

> and I would mention the difference with jdbc/reducible-query in the docstring, I'm sure people will have the same question as I did.

I have included this already, though not in such detail:

https://github.com/metabase/metabase/blob/2d5975e9ea8d3d1ee7af755be34d9340cc06fbc5/src/metabase/driver/sql_jdbc/execute.clj#L718-L719